hrl/__main__.py

```diff
@@ -44,12 +54,15 @@
     if args.clear_option_buffers:
         assert not args.use_global_value_function
 
-    if args.environment == "umaze":
-        mdp = D4RLAntMazeMDP("umaze", goal_state=np.array((0, 8)), seed=args.seed)
+    if args.environment in ["antmaze-umaze-v0", "antmaze-medium-play-v0", "antmaze-large-play-v0"]:
+        env = gym.make(args.environment)
+        env = D4RLAntMazeWrapper(env, start_state=((0, 0)), goal_state=np.array((0, 8)), use_dense_reward=args.use_dense_rewards)
+        seeding.seed(0, random, torch, np)
```

Comment:
I think we can. 

In Cam's codebase, he used it this way -- only using the user-specified seed for the gym environment. I'm not sure what's the reasoning behind it. I think the reason is that when plotting with different random seeds, the neural network is still initialized the same way because the torch seed is the same, and so the randomness only comes from the gym environment.